interface-spec.md

```diff
@@ -18,17 +18,19 @@
 
 **Properties:**
 
-- `String .value` is refined by each interface which extends Node
+- None 
 
 **Methods:**
 
-- `.equals(Node other)` returns true if and only if the argument is a) of the same type b) has the same contents (value and, if applicable, type or language)
+- `.sameTerm(Node other)` returns true if and only if the argument is a) of the same type b) has the same contents (value and, if applicable, type or language)
 
-### IRI extends Node
+### Symbol extends Node
+
+A node named with a URI.  The node is not a URI itsef.
```

Comment:
In any case, [`Symbol` is an ES6 name](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) so we should avoid using that name in a library. While I can see the argument that _“IRI ≠ node with that IRI”_, all RDF specs use the term “IRI” in practice. It might consequently be the term that is easiest to use for developers.

The important questions are: is the difference between IRI and Node important for JavaScript libraries, how does it affect developers, and which choice is most practical?